sdk/azidentity/managed_identity_credential_test.go

```diff
@@ -2,40 +2,22 @@ package azidentity
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"testing"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
 )
 
-func Test_GetToken_CloudShell(t *testing.T) {
+func TestManagedIdentityCredential_GetTokenInCloudShell(t *testing.T) {
 	managedClient, err := NewManagedIdentityCredential("", newDefaultManagedIdentityOptions())
 	if err != nil {
 		fmt.Println("Managed ID error: ", err)
 	} else {
-		managedAT, err := managedClient.GetToken(context.Background(), azcore.TokenRequestOptions{Scopes: []string{"https://storage.azure.com"}})
+		managedAT, err := managedClient.GetToken(context.Background(), azcore.TokenRequestOptions{Scopes: []string{scope}})
 		if err != nil {
 			fmt.Println("Error: ", err)
```

Comment:
`t.Fatalf()`?

Reply:
Yes, but this is one of those tests that must be run manually in the environment at the moment. So I left if with a Println. 

What I could do is check if the MSI_endpoint is set and handle testing depending on the environment we're in, do you think that would work for testing? 

I'd basically add a check for the endpoint and the result of the test would change depending on where it's available or not.

Reply:
Ah I see.  If we don't want this test to run when that env var is set then you can call `t.Skip()` to skip running the test.